docs/tutorial/tutorial-5/android.rst

```diff
@@ -2,9 +2,234 @@
 Tutorial 5 - Taking it mobile: Android
 ======================================
 
-To deploy our applicaton to Android...
+The process of deploying an application to Android is very similar to the
+process for deploying as a desktop application. Briefcase handles installing
+dependencies for Android, including the Android SDK, the Android emulator,
+and the AdoptOpenJDK Java compiler and runtime.
 
-**Coming soon.**
+Create an Android app and compile it
+====================================
+
+Firs, run the ``create`` command. This downloads an Android app template and
+adds your Python code to it.
+
+.. tabs::
+
+  .. group-tab:: macOS
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase create android
+
+      [helloworld] Generating application template...
+      Using app template: https://github.com/beeware/briefcase-android-gradle-template.git
+      ...
+      [helloworld] Installing support package...
+      ...
+      [helloworld] Installing dependencies...
+      ...
+      [helloworld] Installing application code...
+      ...
+      [helloworld] Installing application resources...
+      ...
+      [helloworld] Application created.
+
+  .. group-tab:: Linux
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase create android
+
+      [helloworld] Generating application template...
+      Using app template: https://github.com/beeware/briefcase-android-gradle-template.git
+      ...
+      [helloworld] Installing support package...
+      ...
+      [helloworld] Installing dependencies...
+      ...
+      [helloworld] Installing application code...
+      ...
+      [helloworld] Installing application resources...
+      ...
+      [helloworld] Application created.
+
+  .. group-tab:: Windows
+
+    .. code-block:: doscon
+
+      (beeware-venv) C:\...>briefcase create android
+
+      [helloworld] Generating application template...
+      Using app template: https://github.com/beeware/briefcase-android-gradle-template.git
+      ...
+      [helloworld] Installing support package...
+      ...
+      [helloworld] Installing dependencies...
+      ...
+      [helloworld] Installing application code...
+      ...
+      [helloworld] Installing application resources...
+      ...
+      [helloworld] Application created.
+
+Once this completes, we'll now have an ``android`` directory in your project.
+This directory will contain a ``Hello World`` folder, which will contain an
+Android project with a Gradle build configuration. This project will contain
+your application code, and a support package containing the Python interpreter
+and the `rubicon-java` library.
+
+In the process, Briefcase downloads the Android SDK. You may be prompted to
+accept Google's Android SDK license.
```

Comment:
Given the file sizes and download times can be considerable, it's worth mentioning that this can take a while. The subsequent warning about Gradle looking stuck can then call back to this warning, mentioning that it's downloading yet more code.

Reply:
Fixed in 0994883